lib/action_subscriber/dsl.rb

```diff
@@ -95,7 +133,11 @@ def _run_action_with_filters(env, action)
       final_block = Proc.new { subscriber_instance.public_send(action) }
 
       first_proc = around_filters.reverse.reduce(final_block) do |block, filter|
-        Proc.new { subscriber_instance.send(filter, &block) }
+        if filter.matches(action)
+          Proc.new { subscriber_instance.send(filter.method, &block) } if filter.matches(action)
```

Comment:
Should this and the around_filters_spec be `:action` as well? Maybe we should use something like `callback` here or `callback_method`. I think action vs method is really ambiguous. Sorry to keep nit-picking here.